test/array.every.js

```diff
@@ -60,7 +60,7 @@ metatests.test('every with two-element arrays', test =>
   )
 );
 
-metatests.test('every', test => {
+metatests.test('every / array', test => {
   const data = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16];
```

Comment:
I think it can be moved to a separate variable.

Reply:
What exactly should I move to separate variable?
```const data = [...]```

Reply:
Yes, `const data = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16]`. I don't mind different name though.

Reply:
Still relevant.

Reply:
ping